core/cloudflow-localrunner/src/main/scala/cloudflow/localrunner/LocalRunner.scala

```diff
@@ -19,18 +19,20 @@ package cloudflow.localrunner
 import java.io.{ Closeable, File, FileOutputStream, OutputStream, PrintStream }
 import java.lang.{ Runtime => JRuntime }
 import java.nio.file._
-
 import scala.concurrent.{ Await, Future }
 import scala.concurrent.duration.Duration
 import scala.util.{ Failure, Success, Try }
 import scala.util.control.NonFatal
 import com.typesafe.config.{ Config, ConfigFactory }
 import org.slf4j.LoggerFactory
 import spray.json._
-import cloudflow.blueprint.deployment.{ ApplicationDescriptor, RunnerConfig, StreamletDeployment, StreamletInstance, Topic }
+import cloudflow.blueprint.deployment.{ ApplicationDescriptor, StreamletDeployment, StreamletInstance, Topic }
 import cloudflow.blueprint.deployment.ApplicationDescriptorJsonFormat._
 import cloudflow.blueprint.RunnerConfigUtils._
 import cloudflow.streamlets.{ BooleanValidationType, DoubleValidationType, IntegerValidationType, StreamletExecution, StreamletLoader }
+import cloudflow.runner
```

Comment:
strange import?

Reply:
ah I see now, rather just use `cloudflow.runner...` instead of `runner...` in code.

Reply:
fixed